app/Resources/doc/import.md

```diff
@@ -10,40 +10,25 @@ Map Building
 Build map via file upload
 -------------------------
 
-The most common way to build a map will likely be uploading a sample file. The system will read the header column
-row and build an initial map. It will try to match any column name in the file with database columns.
+The most common way to build a map will likely be uploading a sample file. The system will read the header column row and build an initial map. It will try to match any column name in the file with database columns.
 
```

Comment:
somewhere between 80-120 characters I would think is fine. If you look at the other ones like the api.md just match that line length.